controllers/base_controller.py

```diff
@@ -117,6 +117,17 @@ def _require_login(self, target_url="/"):
                 abort=True
             )
 
+    def _require_permission(self, permission):
+        self._require_login()
+        logging.info("logged in")
+        self._require_registration()
+        logging.info("registered")
+        if permission not in self.user_bundle.account.permissions:
+            return self.redirect(
```

Comment:
Is redirecting to the homepage the thing we want to do here? Not saying it's not, just wondering what other options could be considered.